webclient/src/App.css

```diff
@@ -39,8 +39,8 @@
   display: inline-grid;
   grid-column-gap: 7px;
   grid-row-gap: 3px;
-  grid-template-columns: auto auto auto auto auto auto auto auto auto auto auto;
-  grid-template-rows:auto auto;
+  grid-template-columns: repeat(25, auto);
```

Comment:
25 for the number of alphabets, however we should prob have this always fixed. 

The layout is determined by the response payload, not the style.